netutils/config/compliance.py

```diff
@@ -97,17 +99,23 @@ def _is_feature_ordered_compliant(feature_intended_cfg, feature_actual_cfg):
     return False
 
 
-def _open_file_config(cfg_path):
+def _open_file_config(cfg_path: str) -> t.Union[str, bool]:
     """Open config file from local disk."""
     try:
         with open(cfg_path, encoding="utf-8") as filehandler:
             device_cfg = filehandler.read()
     except IOError:
-        return False
+        return False  # This should probably be changed to a exception raise. Causing mypy issues on L183, L184
     return device_cfg.strip()
```

Comment:
+1 on the exception part. The return type for this looks weird and would introduce lots of unnecessary checks because what the user of this function probably wants is the `str`. If something points to an exception not being correct here, how about an empty string?